src/Frame.h

```diff
@@ -82,7 +82,7 @@ class Frame {
 	bool HasRotFrame() const { return m_flags & FLAG_HAS_ROT; }
 
 	FrameId GetParent() const { return m_parent; }
-	FrameId GetNonRotFrame() { return IsRotFrame() ? m_parent : m_thisId; }
+	FrameId GetNonRotFrame() const { return IsRotFrame() ? m_parent : m_thisId; }
 	FrameId GetRotFrame() { return HasRotFrame() ? m_children.front() : m_thisId; }
```

Comment:
Is there a reason `GetRotFrame()` isn't marked `const` as well? I'd recommend marking it const if there's nothing preventing that.

Reply:
Done!